v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset_test.py

```diff
@@ -130,6 +135,136 @@ def test_update_vat_with_updated_rdc_snv_indel_38(
                 updates=hl.empty_set(hl.tstruct(callset=hl.tstr, project_guid=hl.tstr)),
             ),
         )
+
+        mock_globals_from_config = Globals(
```

Comment:
is it possible to reduce the "real-ness" of this mock at all?  It's basically just the real config.

Reply:
yeah I need to think a bit more about this, 
I want to mock the `Globals` created from `from_dataset_configs()` because 
- the data in the tables may (and does rn) diverge from what's in the config and we don't want to have to update our test tables if we update something in the configs?
- `from_dataset_configs()` also reads in the dataset raw table which is why I lean towards just mocking the return value of that fn

Reply:
yeah, is one option copying the config as is and changing the one two things that the tests require to be different?

Reply:
ok I just mocked the `get_datasets_to_update()` call to return an empty list. these tests are really testing the `update_table()` functionality